uvplot/io.py

```diff
@@ -102,6 +113,8 @@ def export_uvtable(uvtable_filename, tb, vis="", split_args=None, split=None,
                     (MStb_name, split_args['vis']))
 
         split(**split_args)
+        # after splitting data is put into "DATA" column
+        datacolumn = "DATA"
```

Comment:
I see why we would need this, but this is risky because overrides the user `datacolumn` value. 
Trying to understand the workflow:
 - user sets `datacolumn='DATA'` and does not split -> this is OK 
 - user sets `datacolumn='CORRECTED_DATA'` and does not split -> this is OK as it doesn't get executed
 - user sets `datacolumn='DATA'` and does a split -> this is OK (if your line 116 is always true, please see my question above)
 - user sets `datacolumn='CORRECTED_DATA'` and does a split -> the split is performed with `datacolumn` specified inside `split_args` (the value of `datacolumn` in the function main scope is not used), then the tmp MS does not have a `CORRECTED_DATA` column and the code would break unless this line 117 is executed, so this is OK.

Therefore, this is OK, but in order to avoid unwanted behaviours I'd add a check that if `datacolumn` and `split_args` are passed then raises an exception and forces the user to provide `datacolumn` only inside `split_args` so that the behaviour is well defined: from a splitted tmp MS table, only `DATA` column is used.

Reply:
Perhaps the best thing to do is set `datacolumn=None` by default and check that 

```
if datacolumn and split_args:
    raise ValueError("Both datacolumn and split_args cannot be specified at the same time")
```

Reply:
That workflow was my intention with the addition that datacolumn='DATA_CORRECTED' is the default, meaning that not specifying datacolumn when also splitting will result in an error. It could just be better to have an error-prompt detailing what went wrong and that you must have datacolumn='DATA' when also splitting.

Reply:
Perfect.